cdk/database/__init__.py

```diff
@@ -62,3 +62,43 @@ def dynamodb_single_table(self):
                                         partition_key=aws_dynamodb.Attribute(
                                             name='PK',
                                             type=aws_dynamodb.AttributeType.STRING))
+
+    def dynamodb_dual_tables(self):
+        """
+        This table design has us use multiple dyanmoDB tables for tracking
+        the visits and the users. Ideally, by separating the two, we can more
+        easily perform data analytics.
+
+        Visits:
+
+        - PK = `{date}`
+        - SK = `{username}`
+
+        Visitors:
+
+        - PK = username
+        - SK = `VISITOR_INFORMATION`
+        """
+        # TODO: This fails on an ID error. What is self.id? Do we need to make
+        # 2 stacks for the database? This just seems a little redundant.
+        self.visits_table = aws_dynamodb.Table(self,
+                                               self.id,
+                                               point_in_time_recovery=True,
+                                               removal_policy=core.RemovalPolicy.RETAIN,
+                                               sort_key=aws_dynamodb.Attribute(
+                                                   name='SK',
+                                                   type=aws_dynamodb.AttributeType.STRING),
+                                               partition_key=aws_dynamodb.Attribute(
+                                                   name='PK',
+                                                   type=aws_dynamodb.AttributeType.NUMBER))
```

Comment:
We might want to think about making this a string so that we can be flexible with what that timestamp looks like. It can be a stringified epoch timestamp if we want, but we could also make it a datetime so that it is more easily human readable.

Reply:
So my logic for leaving the PK as a number was that if we wanted to convert it to a human-readable format, we would be probably doing that in Athena anyways when the data is being prepared for export.